mkdocs_awesome_pages_plugin/tests/e2e/test_order_and_sort.py

```diff
@@ -0,0 +1,255 @@
+from .base import E2ETestCase
+
+
+class TestOrder(E2ETestCase):
```

Comment:
Please update the name of the class to match the new filename

Reply:
Done